test/dotnet-workload-install.Tests/WorkloadSetVersionMappingTests.cs

```diff
@@ -48,7 +49,7 @@ public static IEnumerable<object[]> WorkloadVersionsData
         [MemberData(nameof(WorkloadVersionsData))]
```

Comment:
Should we include an item in the member data for version that contain `rtm` since that's one of the special cases handled by `SdkFeatureBand`?

Reply:
Done